frontend/interfaces/UserTypes.ts

```
@@ -21,6 +21,10 @@ export interface PublicUserData {
   bio: string;
 }
 
+export interface ProtectedUserData {
+  
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)